rust/parquet/src/data_type.rs

```diff
@@ -95,6 +96,12 @@ impl From<Vec<u32>> for Int96 {
     }
 }
 
+impl fmt::Display for Int96 {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "{:?}", self.data())
```

Comment:
Minor: Maybe as a followup this should print the number rather than its raw bytes?